tests/OSPSuite.Starter/Presenters/ChartTestPresenter.cs

```diff
@@ -182,10 +182,6 @@ public void ClearChart()
          using (_chartUpdater.UpdateTransaction(Chart))
          {
             ChartEditorPresenter.RemoveDataRepositories(_dataRepositories);
-            _dataRepositories.Each(repository =>
-            {
-               Chart.RemoveCurvesForDataRepository(repository);
```

Comment:
This is now done when removing datarepositories. Anything else does not make sense